config.rb

```diff
@@ -178,6 +213,26 @@
   )
 end
 
+proxy(
+  "/scuole_in_chiaro.html",
+  '/school_in_clear.html',
+  locals: {
+    documents: document,
+  },
+  ignore: true,
+)
+
+documents.each do |documents|
```

Comment:
Vogliamo avere una pagina per ogni documento?

Reply:
No pensavo che funzionasse come project, una card per ognuno

Reply:
Attento anche che nel caso in cui vogliamo avere una pagina sola, come penso si debba fare, non serve iterare sui documents ma basta fare il proxy del solo template passandoli i documents.

Reply:
> No pensavo che funzionasse come project, una card per ognuno
Abbiamo un wireframe per la pagina del singolo documento?

Reply:
Quindi basta che utilizzo solo proxy?

Reply:
Esatto e attento a passare `documents: documents`.